compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/ImageComposeSceneTest.kt

```diff
@@ -94,13 +95,17 @@ class ImageComposeSceneTest {
 
     @Test
     fun `run dialog in center`() {
-        val image = renderComposeScene(
+        val image = ImageComposeScene(
             width = 80,
             height = 40,
         ) {
             Dialog(onDismissRequest = {}) {
                 Box(Modifier.size(20.dp).background(Color.Red))
             }
+        }.use {
+            it.render()
+            // Skip animation for the Dialog appearance
+            it.render(500.milliseconds)
```

Comment:
Please use `hasInvalidations` API in a loop instead of just hardcoding timeout value